plugins/Android/src/net/gree/unitywebview/CWebViewPlugin.java

```diff
@@ -222,24 +222,36 @@ public boolean shouldOverrideUrlLoading(WebView view, String url) {
             }
             mWebViewUA = webSettings.getUserAgentString();
             webSettings.setSupportZoom(true);
-            webSettings.setBuiltInZoomControls(true);
+            webSettings.setBuiltInZoomControls(false);
             webSettings.setDisplayZoomControls(false);
             webSettings.setLoadWithOverviewMode(true);
             webSettings.setUseWideViewPort(true);
+            webSettings.setAppCacheEnabled(true);
             webSettings.setJavaScriptEnabled(true);
+            webSettings.setLayoutAlgorithm(WebSettings.LayoutAlgorithm.SINGLE_COLUMN);
+            webSettings.enableSmoothTransition();
+            //webSettings.setCacheMode(WebSettings.LOAD_NO_CACHE);
             if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN) {
                 // Log.i("CWebViewPlugin", "Build.VERSION.SDK_INT = " + Build.VERSION.SDK_INT);
                 webSettings.setAllowUniversalAccessFromFileURLs(true);
             }
-            webSettings.setDatabaseEnabled(true);
+            //webSettings.setDatabaseEnabled(true);
             webSettings.setDomStorageEnabled(true);
-            String databasePath = webView.getContext().getDir("databases", Context.MODE_PRIVATE).getPath();
-            webSettings.setDatabasePath(databasePath);
+            //String databasePath = webView.getContext().getDir("databases", Context.MODE_PRIVATE).getPath();
+            //webSettings.setDatabasePath(databasePath);
 
+            webView.setScrollBarStyle(View.SCROLLBARS_INSIDE_OVERLAY);
+            webView.setScrollbarFadingEnabled(true);
             if (transparent) {
                 webView.setBackgroundColor(0x00000000);
             }
 
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.KITKAT) {
```

Comment:
Our app uses hardware acceleration, declared in the manifest and thus I tried activating hardware acceleration on the view